### PR TITLE
add thickness parameter to convert 2D object to 3D

### PR DESCRIPTION
**Pull request by francoisbonneau**

### CODE REVIEW

File: src/geode/model/helpers/convert_brep_section.cpp [hunk 1]
```diff
@@ -121,23 +121,59 @@ namespace geode
...
         }
         copy_unique_vertices( section, builder, mappings );
         return std::make_tuple( std::move( brep ), std::move( mappings ) );
     }
+
+    std::tuple< BRep, ModelCopyMapping > extrude_section_to_brep(
+        const Section& section,
+        index_t axis_to_add,
+        double min_axis_coordinate,
+        double thickness )
```
Comment (francoisbonneau) [inline src/geode/model/helpers/convert_brep_section.cpp:151]: Should we use min and max values? or min value and extrusion thickness.

**Comment by francoisbonneau** [comment]
Should we do so many changes in the API?

File: src/geode/model/helpers/convert_brep_section.cpp (hunk 1 above)
Comment (panquez) [reply src/geode/model/helpers/convert_brep_section.cpp:151]: Maybo a top and bottom depth is a good idea

Comment (francoisbonneau) [reply src/geode/model/helpers/convert_brep_section.cpp:151]: bottom and top are maybe too specific.

**Pull request closed without merging**